wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationTableViewDataSource.swift

```diff
@@ -480,8 +480,19 @@ extension ConversationTableViewDataSource {
         // 45 minutes
         let significantTimeInterval: TimeInterval = 60 * 45
         let isTimeIntervalSinceLastMessageSignificant: Bool
+
+        // 1 minute
+        let oneMinuteTimeInterval: TimeInterval = 1 * 60
+        let isTimeIntervalSinceLastMessageOneMunute: Bool
+
         let previousMessage = messagePrevious(to: message, at: index)
 
+        if let timeInternalToPreviousMessageIsOneMin = timeIntervalToPreviousMessage(from: message, at: index) {
+            isTimeIntervalSinceLastMessageOneMunute = timeInternalToPreviousMessageIsOneMin > oneMinuteTimeInterval
+        } else {
+            isTimeIntervalSinceLastMessageOneMunute = false
+        }
```

Comment:
Typo `isTimeIntervalSinceLastMessageOneMunute` and strange name `timeInternalToPreviousMessageIsOneMin` because it holds a time interval and not a bool

Reply:
Fixed!